things/clients/service.go

```diff
@@ -16,15 +16,21 @@ import (
 )
 
 const (
-	MyKey             = "mine"
-	thingsObjectKey   = "things"
+	MyKey = "mine"
```

Comment:
Is there a reason to export this (except for tests)? If not, please don't export.